addons/interact_menu/XEH_preInit.sqf

```diff
@@ -8,6 +8,10 @@ PREP_RECOMPILE_END;
 
 #include "initSettings.sqf"
 
+if (!hasInterface) exitWith { ADDON = true; };
```

Comment:
Why did noone think of this before. This is so trivial and logical..

Reply:
Yeah we have.

Reply:
> Why did noone think of this before. This is so trivial and logical..

maybe because it didn't really hurt?